### DISCUSSION TITLE
Contribute

### DISCUSSION THREAD

**Discussion by reason2call** [phnxdrgn536/FLSun-v400#16] [General]
Hi, 
You ask me to go there for contributing on the project. 
Can you tell me how can I help? 
I saw that the project is mainly focused on windows users, being a Mac user myself, I would have liked to contribute to make the project automatically usable on Mac OS. 
How can I participate? 
See you soon, 

Assad

**Comment by phnxdrgn536** [comment]
I am sadly lacking in my abilities for mac programing. if you are able to do any programs for the mac and would like to build an automatic loader i will be putting the CPL project folder on the github soon. if your not able to do that you can point me in the right direction by giving me the file structure of macs... such as for windows you can either use C:\Users\"username"\AppData\Roaming\cura\5.x or %appdata%\cura\5.x to locate the cura configurations folder. what is it for mac?

**Reply by reason2call** [reply]
I have never developed applications for Mac Os before but it could be an interesting challenge. 
I'll see if it can fit in my skills. 
As for the installation directory, it is for example: "/Users/{username}/Library/Application Support/cura/5.2/" for cura 5.2. 
where {username}  is the current mac user.

I'll see what I can do and come back to the topic. 

Maybe make a script that would update the profiles directly? 

I tell you again. 

See you soon

**Reply by phnxdrgn536** [reply]
yeah i just did some research... turns out i wont be able to do any Mac programming since the only programs to do that only run on macs lol... (thanks a lot apple) since i don't have a mac i cant do anything on this end... maybe find some mac uses that can program to help out... i put the project file and source code for CPL on here now. also take a look at chat.openai.com its the AI that helped me write the program. its pretty awesome.
